src/cirrus/release.py

```diff
@@ -319,6 +334,36 @@ def build_release(opts):
     return build_artifact
 
 
+def update_requirements(path, versions):
+    """
+    _update_requirements_
+
+    Update requirements.txt with the provided versions list, where each item in
+    such a list looks exactly like a version specifier in a requirements.txt
+    file.  Items without '==' are ignored.
+
+    Example:
+    ['foo==0.0.9', 'bar==1.2]
+    """
+    LOGGER.info('Updating {}'.format(path))
+
+    with open(path, 'r+') as fh:
+        # original requirements.txt it its pristine order
+        reqs = OrderedDict(tuple(pkg.split('=='))
```

Comment:
This should work for now, but if we end up with flexible requirements like >= we may need an update here to support the different operations. Suggest merging this as-is for now, but open an issue to support it in the future